strawberry_django/utils.py

```diff
@@ -85,15 +85,29 @@ def is_similar_django_type(a, b):
     return True
 
 
+# dirty workaround, but similar to the one of dataclasses
+# it would be better to use dataclasses.fields
+def _is_not_classvar(annotation, namespace):
```

Comment:
I usually don't like negative checks because it can generate code like `if not _is_not_classvar(...)` to check of it is.

I think it is better to define this as `_is_classvar` and check for `not _is_classvar` below